src/pages/GetInvolved/GetInvolved.js

```diff
@@ -1,42 +1,46 @@
 import React from "react";
-import { useSelector } from "react-redux"
+import { useSelector } from "react-redux";
 import Layout from "components/Layout/Layout";
 import { Link } from "react-router-dom";
-import './GetInvolved.scss';
+import "./GetInvolved.scss";
 
 const GetInvolved = () => {
+  const clubDataAll = useSelector((state) => state.clubs);
 
-  const clubDataAll = useSelector(state => state.clubs)
-
-  const renderClubs = clubs => {
-    return clubs.map((club) => {
-      return (
-        <li>
-          <Link className="get-involved__items--link" key={club.id} to={club.link}>
-            {club.title.rendered}
-          </Link>
-        </li>
-      )
-    })
-  }
+  const renderClubs = clubDataAll.map((club) => {
+    return (
+      <li>
+        <Link className="get-involved__item-link" key={club.id} to={club.link}>
+          {club.title.rendered}
+        </Link>
+      </li>
+    );
+  });
 
   return (
     <Layout>
       <div className="get-involved__container">
-          <div className="get-involved__title">
-            <h2>HUOA Club Listing</h2>
-          </div>
-          <div className="get-involved__description">
-            <p2>Please contact HUOA if you are interested in joining and finding out more about the HUOA clubs.  Contact us at 676-5400.</p2>
-          </div>
+        <div className="get-involved__title">
+          <h2>HUOA Club Listing</h2>
+        </div>
+        <div className="get-involved__description">
+          <p>
+            Please contact HUOA if you are interested in joining and finding out
+            more about the HUOA clubs. Contact us at 676-5400.
+          </p>
+        </div>
         <div className="get-involved__body">
-          <ul className="get-involved__items">
-            {renderClubs(clubDataAll)}
-          </ul>
+          {clubDataAll.length < 1 ? (
+            <h4>Loading Clubs...</h4>
+          ) : (
+            <ul className="get-involved__items">
+              {renderClubs}
+            </ul>
+          )}
```

Comment:
instead of doing a length lookup i think we can just do something like: 

```
          {!!clubDataAll ? ( // This returns a bool - if the variable is defined or undefined. Doing a .length is a bigger toll on memory and we can save a bit by doing this instead
            <h4>Loading Clubs...</h4>
          ) : (
            <ul className="get-involved__items">
              {renderClubs}
            </ul>
          )}
```